src/helpers/AppRoutes.tsx

```diff
@@ -1,12 +1,18 @@
 import React from "react";
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
 import App from "../App";
+import Login from "../auth/Login";
+import { AppProvider } from "./app-store";
 
 const router= createBrowserRouter ([
   { 
     path: "/",
-    element: <App/>  
+    element: <AppProvider> <App/> </AppProvider>  
```

Comment:
please move provider out of this file, and put it inside root component, index.tsx